app/src/controllers/userController.ts

```diff
@@ -0,0 +1,60 @@
+import mongoose from 'mongoose';
+import { UserSchema } from '../models/userModel';
+import { Request, Response } from 'express';
+
+const User = mongoose.model('User', UserSchema);
+
+export class UserController {
+
+    public async index (req: Request, res: Response) {
```

Comment:
all controller "actions" needs annotation, arguments description via type and purpose